### PR TITLE
Arrowhead

### PR DESCRIPTION
**Pull request by jklemen**

### CODE REVIEW

File: newfile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello, world
```
Comment (djkotowski) [inline newfile.txt]: Make sure your branches are clean.

File: arrowhead/arrowhead.rb [hunk 2]
```diff
@@ -15,19 +14,11 @@ class Arrowhead
...
-      if shapes.include? shape
-        arrowhead = shapes[shape]
-        "You have a(n) '#{arrowhead}' arrowhead. Probably priceless."
-      else
-        raise "Unknown shape value. Are you sure you know what you're talking about?"
-      end
-    else
-      raise "Unknown region, please provide a valid region."
-    end
+    raise "Invalid region." unless CLASSIFICATIONS.include? region
```
Comment (djkotowski) [inline arrowhead/arrowhead.rb]: Wrap arguments in parentheses.

**Comment by djkotowski** [comment]
Nice work, though you took away the sass. :disappointed: 

You're done with this one.

**Comment by jklemen** [comment]
Gotta fit it under 80 chars :P

**Pull request closed without merging**